ui/app/controllers/jobs/index.js

```diff
@@ -37,54 +33,16 @@ export default class JobsIndexController extends Controller {
   queryParams = [
     'cursorAt',
     'pageSize',
-    // 'status',
     { qpNamespace: 'namespace' },
-    // 'type',
-    // 'searchTerm',
+    'filter',
```

Comment:
Gave a lot of thought to how this should work, and landed (after much discussion) on "Show the literal filter in the URL and translate it in the controller", instead of "Show simulacra in the URL and parse them into a real filter expression at network request time"

The main benefit here, I hope, is users learn about Filter Expressions.